packages/git/src/common/git-model.ts

```diff
@@ -109,6 +109,7 @@ export namespace GitFileStatus {
             case GitFileStatus.New: return !!staged ? nls.localize('theia/git/added', 'Added') : nls.localize('theia/git/unstaged', 'Unstaged');
             case GitFileStatus.Renamed: return nls.localize('theia/git/renamed', 'Renamed');
             case GitFileStatus.Copied: return nls.localize('theia/git/copied', 'Copied');
+            // eslint-disable-next-line @theia/localization-check
```

Comment:
Is this the solution for things that _are_ VSCode but _aren't_ in our JSON (yet)?

Reply:
Well, in this particular case the `Modified` value seems to exist somewhere in the `nls.metadata.json` (our version). However, I worry about semantics here, since the `Modified` that exists inside of vscode is used for modified preferences. I assume that this particular context does not translate correctly into other language when used for git related modifications.

Also you perhaps have noticed the key to `vscode.git`, which is the builtin extension, that is also translated in the language pack. As this isn't localized text from the actual vscode codebase, it doesn't appear inside of the `nls.metadata.json`. Now that we use `localizeByDefault` basically everywhere, I would be fine with using `theia/*` keys for these instead of relying on the extension translations. There are only around ~20 instances of these cases in Theia. WDYT?

Reply:
I guess my target would be a one-tiered world: everything uses `localizeByDefault`. If we find that the thing translated doesn't exist in the JSON, there are two possible causes: it's a typo / rephrase, or it hasn't been translated by language packs. In the former case, we should correct it, because then it will be translated correctly. In the latter case, rewriting `localizeByDefault` to `localize('some/theia/key')` doesn't actually get us any closer to getting the thing translated, so I don't see the point. Instead, whenever we are able to create new translations ourselves, we can traverse the code base, find any uses of `localizeByDefault` that aren't currently working, translate them, and add it to the JSON (creating an automatic key, if that's necessary to make things work, something like `theia/<package>/<file>/<first word or two as necessary>`). If we want a system that requires investigating the actual origin of a particular string to determine its key and checking the JSON to see if that source is covered, then we'll need a lot more documentation about when and how to do that.

Reply:
Interesting approach. I like the simplicity (for the user, hidden behind a bit of technical complexity). I'll create a draft for this, and add it as an additional commit to this PR.